modules/administration/pages/public-cloud.adoc

```diff
@@ -0,0 +1,351 @@
+[[public-cloud]]
+= Public Cloud
+
+
+Some public cloud environments provide images for {productname} Server and Proxy.
+This section discusses what you will require to run {productname} in a public cloud, and how to set up your installation.
+
+[NOTE]
+====
+Public clouds provide {productname} under a Bring Your Own Subscription (BYOS) model.
+This means that you must register them with the {scc}.
+For more information about registering  {productname} with {scc}, see xref:installation:general-requirements.adoc[].
+====
+
+Depending on the public cloud network you are using, you can locate the {productname} installation images by searching for the  keywords [package]``suse``, [package]``manager``, [package]``proxy``, or [package]``BYOS``.
+
+== Instance Requirements
+
+Select a public cloud instance that meets the hardware requirements in xref:installation:hardware-requirements.adoc[].
+
+In addition, be aware of these important considerations:
+
+* The {productname} setup procedure performs a forward-confirmed reverse DNS lookup.
+This must succeed in order for the setup procedure to complete successfully and for {productname} to operate as expected.
+Therefore, it is important that the hostname and IP configuration be performed prior to running the {productname} setup procedure.
+* {productname} Server and Proxy instances are expected to run in a network configuration that provides you control over DNS entries, but cannot access the wider internet.
+Within this network configuration DNS resolution must be provided, such that `hostname -f` returns the FQDN.
+DNS resolution is also important for connecting clients.
+DNS is dependent on the cloud framework you  choose, refer to the cloud service provider documentation for detailed instructions.
+* We recommend that you locate software repositories, the server database, and the Proxy squid cache on an external virtual disk.
+This prevents data loss if the instance is unexpectedly terminated.
+Instructions for setting up an external virtual disk are contained in this section.toring the database and the repositories on an external virtual disk ensures that the data is not lost should the instance need to be terminated for some reason.
```

Comment:
It seems what's trailing the first sentence is some sort of left-over and should be removed.

Reply:
Good spot!